pom.xml

```diff
@@ -14,6 +14,14 @@
         <testng.version>6.8</testng.version>
     </properties>
 
+    <!-- TODO
+            1. .gitignore is missed
+            2. project does not compiled
+            3. Maven profiles is missed
+            4. Suites is missed
+            5. IDE related files and folders should not being under the git
```

Comment:
I added the following line into the gitignore file
.idea/
But looks like I miss something. The folder .idea/ does not get ignored. What could be the cause?